boucle.py

```diff
@@ -4,18 +4,19 @@
 
 import jack
 import sys
-from clip import Clip, Song
+from clip import Clip, Song, getDefaultOutputNames
 from gui import Gui
 from PyQt5.QtWidgets import QApplication
 from queue import Empty
 
-song = Song(8, 8)
+song = Song(8, 8, getDefaultOutputNames(8))
 
 client = jack.Client("Super Boucle")
 midi_in = client.midi_inports.register("input")
 midi_out = client.midi_outports.register("output")
-outL = client.outports.register("output_L")
-outR = client.outports.register("output_R")
+outL = client.outports.register("MAIN_L")
+outR = client.outports.register("MAIN_R")
+
 inL = client.inports.register("input_L")
 inR = client.inports.register("input_R")
 
```

Comment:
Main port should not be removed like other port so if a song does not use Main port it should remain on port list, keeping connection alive for Main port. This will happen when you create a new empty song : no clip use Main port !
I would like to keep superboucle as simple as possible for beginner. On the other hand, superboucle should be usable in live or with complex setup (routing, effect, tempo map/change) so I agree with your request of "stable" ports setup.

Maybe we can finish port implementation as I suggest in order to quickly close this PR and add all your feature in main branch. Then, we can make another PR to improve port management ? (I also want to add input ports)